apps/angular/projection/src/app/component/city-card/city-card.component.ts

```diff
@@ -1,13 +1,43 @@
 import { Component, OnInit } from '@angular/core';
+import { CityStore } from '../../data-access/city.store';
+import { FakeHttpService } from '../../data-access/fake-http.service';
+import { CardType } from '../../model/card.model';
+import { City } from '../../model/city.model';
+import { ListItem } from '../../model/listItem.model';
+import { CardComponent } from '../../ui/card/card.component';
 
 @Component({
   selector: 'app-city-card',
-  template: 'TODO City',
+  template: `
+    <app-card
+      [list]="listToSend"
+      [type]="cardType"
+      [color]="'rgba(0,0,255,0.1)'"></app-card>
+  `,
   standalone: true,
-  imports: [],
+  imports: [CardComponent],
 })
 export class CityCardComponent implements OnInit {
-  constructor() {}
+  cities: City[] = [];
+  cardType = CardType.CITY;
+  listToSend!: ListItem[];
 
-  ngOnInit(): void {}
+  constructor(
+    private http: FakeHttpService,
+    private store: CityStore,
+  ) {}
+
+  ngOnInit(): void {
+    this.http.fetchCities$.subscribe((c) => this.store.addAll(c));
+    this.store.city$.subscribe((c) => {
```

Comment:
avoid imperative coding, prefer declarative using Rxjs or Signals. (But prefer Signals when it comes to template binding)